redisinsight/ui/src/components/query/query-card/QueryCardCommonResult/components/CommonErrorResponse/CommonErrorResponse.tsx

```diff
@@ -33,7 +34,7 @@ const CommonErrorResponse = (id: string, command = '', result?: any) => {
   }
   // Flow if SUBSCRIBE command was executed
   if (checkUnsupportedCommand([CommandSubscribe.toLowerCase()], commandLine)) {
-    return cliTexts.SUBSCRIBE_COMMAND(Pages.pubSub(instanceId))
+    return <UsePubSubLink path={Pages.pubSub(instanceId)} />
```

Comment:
- by naming convention in React only hooks should start with prefix use
- do we need this component at all? Can we use FeatureFlagComponent with otherwise prop?

Reply:
In my opinion, all output should be done via components, also we already have `UseProfilerLink` component, which is equivalent to this one. 
Use convention is for function calls, not components, but provided that we decide to change all similar components to different convention, I am OK with this as well

Reply:
you mean output of unsupported commands? There is an object of small functions or texts, why do we need to create components for each?

Reply:
@zalenskiSofteq why not? This is supposed to be declarative react app. Otherwise, why not just use html strings?
Having the display logic related to specific text encapsulated in component is the best way. If you have 100 components like this, then probably there is something wrong with the architecture

Reply:
@pd-redis 
I don’t understand, please explain more clearly. Do you think creating 100 components for each comand is good or bad?

I wrote:

> “We shouldn’t create 100 components.”

You replied:

> “Why not?”

 — this means you think it’s good.

But then you wrote, 

> there is something wrong with the architecture.

Reply:
If it makes sense for the use cases, yes I think it is good. In this case, there is some logic, that needs to be performed, if we apply it inline for every potential case, that would make CommonErrorResponse.tsx file even harder to parse. Also extra benefit of separate component is ability to test it in isolation, which is not present with CommonErrorResponse approach.

Reply:
To test it in isolation and apply the DRY principle, it's better to create a single generic component for such commands.